services/QuillLMS/client/app/bundles/Grammar/styles/style.scss

```diff
@@ -3,11 +3,11 @@
 @import "./node_modules/bulma/bulma.sass";
 @import './admin.scss';
 @import "./admin-regex-rules.scss";
-@import './selectSearch.scss';
 @import './dashboards.scss';
 @import './intro.scss';
 @import './headerStyling.scss';
 @import '../../../../../app/assets/stylesheets/shared/teacher-preview-menu.scss';
+@import "../../../../node_modules/react-select-search/style.css";
```

Comment:
I wonder if, since we use this in three apps, it makes sense to import this into the Shared bundle's CSS file instead?

Reply:
That makes sense! I'll update